recipes/gluonts/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "gluonts" %}
+{% set version = "0.11.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gluonts-{{ version }}.tar.gz
+  sha256: c5db67304853db450620edf2a5bf248eb7e6ee65e4290641bf725e4a2e7d00db
+
+build:
+  skip: true  # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - sphinx
+    - click
+    - mypy
+    - black
+    - pip
+  run:
+    - python
+    - numpy >=1.16,<2.dev0
+    - pandas >=1.0,<2.dev0
+    - pydantic >=1.7,<2.dev0
+    - tqdm >=4.23,<5.dev0
+    - toolz >=0.10,<1.dev0
+    - typing-extensions >=4.0,<5.dev0
+    - dataclasses  # [py==36]
+
+test:
+  imports:
+    - gluonts
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/awslabs/gluonts/
+  summary: GluonTS is a Python toolkit for probabilistic time series modeling, built around MXNet.
+  license: Apache-2.0
+  license_file:
+    - NOTICE
+    - LICENSE
```

Comment:
The copies of these files can be removed here.

By referencing them under `license_file` conda will automatically include copies from the source at build time.